developing/cycle/tracker.md

```diff
@@ -4,5 +4,77 @@
 
 We use Pivotal Tracker to track tasks and estimate how much time each task will take.
 
-* 🔒[How to organize a Pivotal project](https://docs.google.com/document/d/1abmxQdYol6Zi3kR3U__JNJl3uPdKF9WPYWfu75AkZ50/edit#heading=h.yyb7wuihugn8)
-* [Pivotal workflow](https://www.pivotaltracker.com/help/articles/workflow_overview/)
+- 🔒[How to organize a Pivotal project](https://docs.google.com/document/d/1abmxQdYol6Zi3kR3U__JNJl3uPdKF9WPYWfu75AkZ50/edit#heading=h.yyb7wuihugn8)
+- [Pivotal workflow](https://www.pivotaltracker.com/help/articles/workflow_overview/)
+
+*Remember, agile processes should be treated [more like guidelines than actual
+rules](https://www.youtube.com/watch?v=jl0hMfqNQ-g).*
+
+## Stories
+
+### State definitions
+
+- Started (waiting to be finished) -- The story is *actively* being worked on.
+
+- Finished (waiting to be delivered) -- The work is believed to be done. It may
+  or may not have passed code review yet.
+
+- Delivered (waiting on acceptance) -- For code, this means it's landed in
+  master and passed the build, or has gone all the way to production, whichever
+  makes sense for the story. For other work, use common sense.
+
+### Creating
+
+- Add enough information so anyone on the team can pick it up.
+
+- Include acceptance criteria (AC) to make it clear what "done" means and to
+  help the person accepting the work know what to validate.
+
+- Leave the story unestimated.
+
+- Add it to the top of the inbox (at the top of icebox).
+
+### Finishing / Delivering
+
+- Add comments, PRs, etc. to the story that will help someone else accept the work.
+
+### Accepting
+
+- Throughout the week, everyone should take time to accept other people's work.
+
+- Look at the acceptance criteria, validate they've been met, and click the
+  green button.
+
+## Weekly planning
+
+Weekly sprints begin with a planning session. It goes like this:
+
+1. Go through in progress stories
+
+   - Determine if the story is blocked.
+
+   - If a story will be blocked for an unknown amount of time, or if it's
+     unlikely it'll be unblocked this week, move it into the "blocked" section
+     of the icebox.
+
+2. Go through "blocked" section of the icebox
+
+   - If a story is no longer blocked, move it into backlog.
+
+   - If a story has been blocked for more than a few weeks, delete it or move it
+     to the bottom of the icebox. Own up to the fact that it's not getting done.
+
+   - If a "really important" story has been blocked for more than a few weeks,
```

Comment:
should there be something here to document our preferences for self blocking vs blocked by another story? this last point here seems to imply the former is our usual case, whereas at least on Reserve we prefer the latter.

Reply:
What is "self blocking"?

Reply:
@brainsik a story that is blocked by something that isn't a story, i.e. writing something like "Get brainsik to explain this to us" instead of a story describing other work

Reply:
This last point is really just about getting folks to own up to what it means for a story to get blocked. If things are blocked for a long time, you either need to accept that it's not getting done and pull it into the icebox, or step up and prioritize the work to unblock it. This is true regardless of how things are blocked (self or story). You shouldn't be processing the same story week after week. For example, you could put into the icebox and update the blocking story to have another AC that the blocked story gets moved to the inbox.